packages/react-client/src/hooks/deviceManagers/audioDeviceManager.ts

```diff
@@ -0,0 +1,9 @@
+import { useFishjamContext } from "../fishjamContext";
+import { useDeviceManager } from "./deviceManager";
+
+export const useAudioDeviceManager = () => {
+  const { audioDeviceManagerRef } = useFishjamContext();
+  const audioDeviceState = useDeviceManager(audioDeviceManagerRef.current);
+
+  return { ...audioDeviceState };
```

Comment:
Why not just:
```suggestion
  return audioDeviceState;
```

Reply:
i think it's a leftover from extending the object by some properties